ui/app/pages/first-time-flow/seed-phrase/seed-phrase-intro/seed-phrase-intro.component.js

```diff
@@ -0,0 +1,113 @@
+// Components
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Button from '../../../../components/ui/button';
+// Routes
+import { INITIALIZE_SEED_PHRASE_ROUTE } from '../../../../helpers/constants/routes';
+
+export default class SeedPhraseIntro extends PureComponent {
```

Comment:
[REQ] We don't want to extend the pattern of using class components and splitting into containers/components. Could you refactor this into a functional component?

Reply:
Force of habit ;) Can do!

Reply:
done